xee/ext.py

```diff
@@ -467,7 +467,7 @@ def image_to_array(
     data = arr.T
 
     # Sets EE nodata masked value to NaNs.
-    data = np.where(data == self.mask_value, np.nan, data)
+    data = np.where(np.isclose(data, self.mask_value), np.nan, data)
```

Comment:
How about explicit specifying `rtol`? That makes it more obvious that the precision was an intentional choice.

Or alternatively, maybe it would suffie to simply cast `mask_value` to `data.dtype`?

Reply:
Hello @shoyer I specified `rtol` value in the code.